.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Build Docker images
+
+on:
+  pull_request:
+  push:
+    branches-ignore: 'main'
+
+jobs:
+  ubuntu:
+    strategy:
+      matrix:
+        arch:
+          - arm64
+          - x86_64
+        distro:
+          - focal
+          - jammy
```

Comment:
Why Jammy? It's not listed in https://github.com/open-quantum-safe/liboqs/blob/main/PLATFORMS.md
Focal is listed, but shouldn't be (if this lands) -- so remove this once PLATFORMS gets updated?

Reply:
My intent was to update PLATFORMS.md to list support for Focal, Jammy, and Noble (i.e., every Ubuntu LTS version still receiving [standard security maintainence](https://ubuntu.com/about/release-cycle)) on x86_64 and arm64. Do we really want to support only the latest LTS? Focal will be receiving security updates for another year and Jammy for another three.

Reply:
That seems reasonable to me.